xmodule/util/xmodule_django.py

```diff
@@ -5,9 +5,39 @@
 """
 
 
+from os import listdir
+
 import webpack_loader
 # NOTE: we are importing this method so that any module that imports us has access to get_current_request
 from crum import get_current_request
+from django.conf import settings
+from webpack_loader.loader import WebpackLoader
+
+
+class XmoduleWebpackLoader(WebpackLoader):
+    """Custom webpack loader.This is where behavior can be customized
+    as to how edx-platform static files are loaded.
```

Comment:
Can you be more specific in this docstring? It is not just any old edx-platform static files which you handle here: it is specifically assets for XModule-style XBlocks.

Reply:
Please check last commit, also I improved the docstring for load_assets method